src/index.mjs

```diff
@@ -14,9 +14,9 @@
 const CACHE = {};
 
 export default function html(statics) {
-	const key = statics;
-
+	const key = statics.length + statics;
```

Comment:
This has a key collision in cases such as `<a>${foo},</a>` and `<a>,${foo}</a>`.